data/plugins/entity/interaction/object.rb

```diff
@@ -0,0 +1,6 @@
+# Catch all object actions and turn the player to
+# the position of the object
+
+on :message, :object_action do |player, message|
+  player.turn_to message.position
```

Comment:
We currently have code for finding the origin and edges of an object, but having 'interactable sides' is news to me.  Got anywhere you can point to for this?